tailwind.config.js

```diff
@@ -29,5 +29,7 @@ module.exports = {
             scale: ['active']
         },
     },
-    plugins: [],
+    plugins: [
+        require('@tailwindcss/aspect-ratio')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bastiengrignon/mazette/pullRequest?prid=7884306)
```suggestion
        require("@tailwindcss/aspect-ratio")
```